style.css

```diff
@@ -0,0 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Poppins, sans-serif;
+}
+
+header .navbar {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
- [ ] Remember that your design should sticks as much as possible to the Figma design 🖌️. Kindly add margins to your design to stick with the original layout 🇿🇲.

Your design: 
![Screenshot from 2022-10-26 20-58-32](https://user-images.githubusercontent.com/91301423/198173318-c080fa12-499a-4c22-8bcc-fc9d05b9ae4f.png)

Figma design:
![image](https://user-images.githubusercontent.com/91301423/198173264-5009c5d3-609b-4665-bf04-7b973225c988.png)

Reply:
I followed the figma design by giving my header a padding-top of 114px and for the left and right padding I gave them 24px.